### ISSUE TITLE
Sanitize CSV headers

### ISSUE THREAD

**Issue by hammady** [daqcri/NADEEF#38]
When receiving CSV files, the header should be sanitized for proper Postgres column names.

**Comment by visenger** [comment]
Hi @hammady , I just stumbled upon the fact that NADEEF FD rules prohibit attributes to have whitespaces? Is this a design choice?

**Comment by hammady** [comment]
Can you please give examples?

**Comment by visenger** [comment]
here is an example

 ```
  {
	   "name" : "FD1",
            "type" : "fd",
            "value" :  [" Paper ID, Author ID | Original affiliation name "] 
        },
```
is considered as not valid. 
I had to preprocess all my data in order to remove the whitespaces in column names, like Paper ID becomes PaperID and so on.

**Comment by hammady** [comment]
@visenger yes, that was assumed for simplicity. You are welcome to open a pull request for a fix. Thanks.
